e2e/etcdctlv3_test.go

```diff
@@ -100,6 +101,56 @@ func testCtlV3Set(t *testing.T, cfg *etcdProcessClusterConfig, dialTimeout time.
 	}
 }
 
+func TestCtlV3Watch(t *testing.T) {
+	testCtlV3Watch(t, &configNoTLS, 5*time.Second, false, false)
+}
+
+func TestCtlV3WatchInteractive(t *testing.T) {
+	testCtlV3Watch(t, &configNoTLS, 5*time.Second, false, true)
+}
+
+func testCtlV3Watch(t *testing.T, cfg *etcdProcessClusterConfig, dialTimeout time.Duration, quorum, interactive bool) {
+	defer testutil.AfterTest(t)
+
+	os.Setenv("ETCDCTL_API", "3")
```

Comment:
this will need to be refactored into something like `ctlV3Test(t, myTestFunc)`; now or later?